src/lib/loaders/loaders_with_authentication/gravity.js

```diff
@@ -138,6 +138,21 @@ export default (accessToken, userID, opts) => {
     ),
     updateMeLoader: gravityLoader("me", {}, { method: "PUT" }),
     orderLoader: gravityLoader(id => `order/${id}`, {}, { headers: true }),
+    confirmOrderLoader: gravityLoader(
+      id => `order/${id}`,
+      { state: "approved" },
+      { method: "PUT" }
+    ),
+    finalizeOrderLoader: gravityLoader(
```

Comment:
@ashkan18 does this map how stress would receive these mutations? If we're talking directly to gravity we could have the three separate mutations but a single loader here that passes in different params from the mutation itself.

Reply:
Stress also has separate mutations for each state, Currently:
- `CreateOrder`
- `SubmitOrder`
- `ApproveOrder`
- `RejectOrder`
and
- `FinalizeOrder`
Lets decide on these names, i can change names on stress mutations too but lets finalize the names internally. Do these names make sense? Is it easy to make changes here? I can easily change names in Stress too if we want other names.